server/routerlicious/packages/routerlicious-base/src/alfred/runnerFactory.ts

```diff
@@ -252,6 +252,9 @@ export class AlfredResourcesFactory implements core.IResourcesFactory<AlfredReso
 			false,
 		);
 
+		const checkpointsTTLSeconds = config.get("checkpoints:checkpointsTTLInSeconds") ?? 864000;
+		await checkpointsCollection.createTTLIndex({ _ts: 1 }, checkpointsTTLSeconds);
```

Comment:
I recalled I read somewhere this will not work if there are some existing data, but I can't find what I read anywhere.

So I think either I understand wrong or it was some legacy information. Feel free to checkin like this, but please take a test with non empty collection and try if this work from code. (Like deploy to a cluster with non empty `checkpoints` collection with this change)

Reply:
According to this, adding the TTL index automatically applies it to existing documents: https://learn.microsoft.com/en-us/azure/cosmos-db/mongodb/time-to-live#:~:text=Once%20the%20index%20is%20created%2C%20the%20database%20will%20automatically%20delete%20any%20documents%20in%20that%20collection%20that%20have%20not%20been%20modified%20in%20the%20last%2010%20seconds.

Reply:
Once the changes were deployed, the documents that were in the database longer than the TTL were removed.